### DISCUSSION TITLE
Custom File Descriptions

### DISCUSSION THREAD

**Discussion by notrespire** [DirectoryLister/DirectoryLister#857] [Ideas & Features]
Would be cool to be able to add a description text to files when opening the information modal.

Is this possible without running a database to reference the files?

**Comment by PHLAK** [comment]
This would be cool but I'm not sure it's possible for _all_ files without persistent storage of some sort.

Perhaps there's a way I could get it to show file metadata if present. For example, audio track information (e.g. album, artist, etc.) or image EXIF data. The downside to this is every file type would require unique code to extract the metadata which would be a chore to implement.

This is the best I can think of at the moment but may explore this further in the future.
